packages/components/modules/activity-log/ActivityLogComponent/index.tsx

```diff
@@ -0,0 +1,62 @@
+'use client'
+
+import { ChangeEventHandler, FC, useState, useTransition } from 'react'
+
+import { Searchbar } from '@baseapp-frontend/design-system'
+
+import { Box, Typography } from '@mui/material'
+import { useForm } from 'react-hook-form'
+
+import SearchNotFoundState from '../../messages/SearchNotFoundState'
```

Comment:
let's avoid importing components from other modules, if it's meant to be shared between modules, we should move it to `/modules/__shared__` and use it from there